docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -252,8 +252,7 @@ workspaces at pathname "root:imw1".
 
 4. Ensure the existence of zero, one, or more workload management workspaces 
 depending on the value of `--ensure-wmw` flag. Default is one workload management
-workspaces at pathname "root:wmw1". The workload management workspaces have APIBindings
-that import the namespaced Kubernetes resources (kinds of objects) for management of
+workspaces at pathname "root:wmw1". The workload management workspaces have CRDs for management of
```

Comment:
This needs more revision due to the move to using the space framework.

Reply:
Could you elaborate on the needed revisions? Thanks!

Reply:
I see. But the change here is aiming at https://github.com/kubestellar/kubestellar/pull/1395#pullrequestreview-1769284569, to remove the APIBinding related stuff. And the PR is meant to address the two comments. Let's do incremental/agile steps toward the current terminologis?

Reply:
OK